### ISSUE TITLE
Test sample by sample

### ISSUE THREAD

**Issue by ak-karimzai** [josebeo2016/BTS-Encoder-ASVspoof#2]
is it possible to test just a sample of audio with pretrained model?

**Comment by nguyenvulong** [comment]
Take a look at this directory https://github.com/josebeo2016/BTS-Encoder-ASVspoof/tree/main/demo2023
You can load the pretrained models and calculate the logits from the input audio.
